mysite/w3/library.py

```diff
@@ -1,9 +1,10 @@
 from math import ceil
 from flask import g as session_data
-from consts import maxStaticBookLevels, maxScalingBookLevels, maxSummoningBookLevels, maxOverallBookLevels, skill_talentsDict, combat_talentsDict, currentWorld
+from consts import maxStaticBookLevels, maxScalingBookLevels, maxSummoningBookLevels, maxOverallBookLevels, skill_talentsDict, combat_talentsDict, currentWorld, stamp_maxes, maxMealLevel 
 from models.models import AdviceSection, AdviceGroup, Advice
 from utils.data_formatting import mark_advice_completed
 from utils.logging import get_logger
+from w4.breeding import parseJSONtoBreedingDict
```

Comment:
Lets remove this unused import for now. I will work on integrating that function into the Account model afterwards during this beta so we can get proper values for the meal effect pets.